README.md

```diff
@@ -254,26 +252,26 @@ collector.close()
 Look at the result saved in tensorboard: (with bash script in your terminal)
 
 ```bash
-tensorboard --logdir log/dqn
+$ tensorboard --logdir log/dqn
 ```
 
 You can check out the [documentation](https://tianshou.readthedocs.io) for advanced usage.
 
 ## Contributing
 
-Tianshou is still under development. More algorithms and features are going to be added and we always welcome contributions to help make Tianshou better. If you would like to contribute, please check out [docs/contributing.rst](https://github.com/thu-ml/tianshou/blob/master/docs/contributing.rst).
+Tianshou is still under development. More algorithms and features are going to be added and we always welcome contributions to help make Tianshou better. If you would like to contribute, please check out [this link](https://tianshou.readthedocs.io/en/latest/contributing.html).
 
 ## TODO
 
-Check out the [Issue/PR Categories](https://github.com/thu-ml/tianshou/projects/2) and [Support Status](https://github.com/thu-ml/tianshou/projects/3) page for more detail.
+Check out the [Project](https://github.com/thu-ml/tianshou/projects) page for more detail.
 
 ## Citing Tianshou
 
 If you find Tianshou useful, please cite it in your publications.
 
 ```latex
 @misc{tianshou,
-  author = {Jiayi Weng, Minghao Zhang, Dong Yan, Hang Su, Jun Zhu},
+  author = {Jiayi Weng, Minghao Zhang, Alexis Duburcq, Kaichao You, Dong Yan, Hang Su, Jun Zhu},
   title = {Tianshou},
```

Comment:
Your way to cite Tianshou is for publication, not software. It is not very appropriate.

Reply:
Yes, we are submitting tianshou to JMLR and we would like to place your name into the authorlist.

Reply:
That would be awesome :smiley: